README.md

```diff
@@ -14,30 +14,34 @@
 
 [![Paypal](https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif)](https://www.paypal.com/donate?business=RCR8HT8GDC5XC&item_name=Free+Software&currency_code=USD)
 
-https://github.com/tilkinsc/LuaConsole  
-A next-gen, Cross-Platform [Lua-5.1.x, LuaJIT-2.0, Lua-5.2.x, Lua-5.3.x, Lua-5.4.x]-supporting CLI made to supersede PUC-Lua and LuaJIT CLI  
+[LuaConsole](https://github.com/tilkinsc/LuaConsole) is a next-gen, Cross-Platform [Lua-5.1.x, LuaJIT-2.0, Lua-5.2.x, Lua-5.3.x, Lua-5.4.x]-supporting CLI made to supersede PUC-Lua and LuaJIT CLI
```

Comment:
This was intentionally designed to dump a link here. If you want this change, please update all the other readme's to this format. Otherwise revert it.

Reply:
Done